build.gradle

```diff
@@ -1,8 +1,8 @@
 plugins {
-  id 'java'
-  id 'org.springframework.boot' version '3.1.1'
-  id 'io.spring.dependency-management' version '1.1.0'
-  id 'org.graalvm.buildtools.native' version '0.9.23'
+    id 'java'
```

Comment:
Please get rid of whitespace changes (I don't think this file changed at all so you could just grab it from main).